app/src/main/java/com/keylesspalace/tusky/fragment/TimelineFragment.java

```diff
@@ -301,20 +301,21 @@ private void updateCurrent() {
                             if (!statuses.isEmpty()) {
                                 filterStatuses(statuses);
 
-
                                 if (!this.statuses.isEmpty() && topId != null) {
                                     // clear old cached statuses
-                                    Iterator<Either<Placeholder, Status>> iterator = statuses.iterator();
+                                    Iterator<Either<Placeholder, Status>> iterator = this.statuses.iterator();
```

Comment:
yeah... this code never actually worked because it ran on the wrong list 🙄

Reply:
That looks like very confusing variable naming. Using mStatuses or something like that for class members would really help avoid this ;)

Reply:
No Hungarian notation will be in the code I write 😜

Reply:
Then at least don't use the same name as a class variable and a local variable :p

Reply:
I would rather rename local `statuses` into something `newStatuses` or `loadedStatuses`